python/graphstorm/model/node_decoder.py

```diff
@@ -65,10 +65,33 @@ def forward(self, inputs):
 
     def predict(self, inputs):
         """ Make prediction on input data.
+
+        Parameters
+        ----------
+        inputs : tensor
+            The input features
+
+        Returns
+        -------
+        Tensor : maximum of the predicted results
+        """
+        logits = th.matmul(inputs, self.decoder)
+        return (th.sigmoid(logits) > .5).long() if self._multilabel else logits.argmax(dim=1)
```

Comment:
Does this ((th.sigmoid(logits)) come from sklearn?

Reply:
We apply sigmoid to make the logits between (0, 1)